samples/kotlin-native/build.gradle.kts

```diff
@@ -13,6 +13,7 @@ kotlinNativeSourceSets["main"].component {
     if (this is KotlinNativeMainComponent) {
         outputKinds.set(listOf(OutputKind.EXECUTABLE))
         baseName.set("foo")
+        extraOpts(listOf("-entry", "org.jlleitschuh.gradle.ktlint.sample.native.main"))
```

Comment:
Why this change?

Reply:
New experimental plugin has not yet supported defining entry point, like "konan" plugin. 

There is open PR: https://github.com/JetBrains/kotlin-native/pull/1996 that adds such support.

Reply:
👍